### PR TITLE
feat: Add order status helper method

### PR DESCRIPTION
**Pull request by yusufttur**
Add order status helper method with pre-defined statuses.

### CODE REVIEW

File: src/services/ShopwareDataHelpers.ts [hunk 1]
```diff
@@ -233,4 +233,9 @@ export const getMediaId = async (fileName: string, adminApiContext: AdminApiCont
 export function extractIdFromUrl(url: string): string | null {
     const segments = url.split('/');
     return segments.length > 0 ? segments[segments.length - 1] : null;
-}
\ No newline at end of file
+}
+
+type OrderStatus = 'cancel' | 'complete' | 'reopen' | 'process';
+export const setOrderStatus = async (orderId: string, orderStatus: OrderStatus , adminApiContext: AdminApiContext): Promise<void> => {
+    await adminApiContext.post(`./_action/order/${orderId}/state/${orderStatus}`);
```
Comment (Phil23) [inline src/services/ShopwareDataHelpers.ts]: Add a return of the API call, so the external user could react to the result of the call.

**Review by Phil23 — CHANGES_REQUESTED** [review]

File: src/services/ShopwareDataHelpers.ts (hunk 1 above)
Comment (yusufttur) [reply src/services/ShopwareDataHelpers.ts:240]: Should I add all the data, including the response from the call, or just the transaction ID?
This is how it looks like:
{
    "extensions": {
        "foreignKeys": {
            "extensions": [],
            "apiAlias": "state_machine_state_foreign_keys_extension"
        },
        "internal_mapping_storage": {
            "extensions": [],
            "apiAlias": null
        }
    },
    "_uniqueIdentifier": "018fe21a11e6718a83122821830392b0",
    "versionId": null,
    "translated": {
        "name": "Cancelled",
        "customFields": []
    },
    "createdAt": "2024-06-04T07:15:10.441+00:00",
    "updatedAt": null,
    "name": "Cancelled",
    "technicalName": "cancelled",
    "stateMachineId": "018fe21a11e6718a8312282180e1dfd2",
    "stateMachine": null,
    "fromStateMachineTransitions": null,
    "toStateMachineTransitions": null,
    "translations": null,
    "orders": null,
    "orderTransactionCaptures": null,
    "orderTransactionCaptureRefunds": null,
    "orderTransactions": null,
    "orderDeliveries": null,
    "fromStateMachineHistoryEntries": null,
    "toStateMachineHistoryEntries": null,
    "customFields": null,
    "id": "018fe21a11e6718a83122821830392b0"
}

Comment (yusufttur) [reply src/services/ShopwareDataHelpers.ts:240]: In other words, is this change good?
```
type OrderStatus = 'cancel' | 'complete' | 'reopen' | 'process';
export const setOrderStatus = async (orderId: string, orderStatus: OrderStatus , adminApiContext: AdminApiContext): Promise<string> => {
    const resp = await adminApiContext.post(`./_action/order/${orderId}/state/${orderStatus}`);
    const result = (await resp.json()) as { data: { id: string }; };
    return result.data.id;
};
```

Comment (yusufttur) [reply src/services/ShopwareDataHelpers.ts]: Return added to API call

**Review by Phil23 — APPROVED** [review]

**Pull request merged**